mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -119,8 +119,7 @@ extern "C" void mgpuMemHostRegisterInt32(int64_t rank, void *ptr) {
   mgpuMemHostRegisterMemRef(desc->data + desc->offset, sizes, strides, 123);
 }
 
-template <typename T>
-void mgpuMemGetDevicePointer(T *hostPtr, T **devicePtr) {
+template <typename T> void mgpuMemGetDevicePointer(T *hostPtr, T **devicePtr) {
```

Comment:
no visible functional change in this file. probably better to leave this one unchanged.

Reply:
That's right, no functional changes. However, our CI enforces clang-format to pass (on code diffs). Moving this file will trigger clang-format check on it. I wouldn't be able to pass CI without making those changes.